dicomsort

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 #############
-#This script calls the afni program dicom_hdr to read header information, reads the information 
-#and then renames and sorts the files alphabetically
+#This script use the afni programs dicom_hdr and dicom_hinfo to read header information, 
+#reads the information and then renames and sorts the files alphabetically, optionally
+#creating subdirectories for each series.
+#Written by Michael Worden
+#Updated 02/07/2024 to add support for optionally creating subject and series 
+#subdirectories
```

Comment:
Consider adding `set -euo pipefail` to the beginning of this file. That command adds some extra error checks that are really helpful

More info here: http://redsymbol.net/articles/unofficial-bash-strict-mode/ (I don't reset the IFS generally unless I have to for reading a file or something)